kaldb/src/main/proto/metadata.proto

```diff
@@ -109,3 +109,25 @@ message RecoveryTaskMetadata {
   // Created timestamp
   int64 created_time_epoch_ms = 5;
 }
+
```

Comment:
Add some documentation for the messages.

Reply:
Revised descriptions for fields to include additional detail.